src/app/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Card, CardContent, Tab, Tabs, Typography } from "@mui/material";
+import React from "react";
+import styles from "./style.module.scss";
+import TableTranslation from "../tableTranslation/TableTranslation";
+import TableDisplayTranslation from "../tableDisplayTranslation/TableDisplayTranslation";
+
+export const TabsComponent = ({ projectsId }: { projectsId: any }) => {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+  return (
+    <>
+      <Tabs
+        value={value}
+        onChange={handleChange}
+        aria-label="basic tabs example"
+        className={styles.Tabs}
+      >
+        <Tab label="Languages" />
+        <Tab label="Translation" />
+      </Tabs>
+
+      <Box>
+        {/* Languages Tab */}
+        {value === 0 && (
```

Comment:
value should not be numbers , when i read it i didn't understand what it stand for , so you should make an Enum with values in an external type file, use it instead of numbers